language/stdlib/modules/LibraAccount.move

```diff
@@ -188,6 +201,25 @@ module LibraAccount {
         );
     }
 
+    public fun initialize_writeset_manager(
```

Comment:
Yesss! There's a slight gotcha here: the order of how event handles are generated are depending on the order of calls in  the genesis module. If we merge those functions, the event key for a number of functions will be modified. The most important one of them is probably the EventKey for new_epoch_event will be off by one, which will be a "bigger" change. Some of the json-rpc tests also hard coded those event keys. That's why I keep the two resource separate and have two init functions so that the order of creating event handles could be perserved.

Reply:
Avoiding churn in the EventKeys seems worth it to have 2 init functions. My $0.02

Reply:
There is already a breaking PR that changes the initialization sequence of events and changes these event keys (see Davids SEV post in libra core). It might be worthwhile to try and coordinate this with that PR, and then we could combine these and have only one break of the event keys?